### PR TITLE
feat: update release info to v5.31.0

### PR DESCRIPTION
**Pull request by barmac**
Related to #4753

### Proposed Changes

For suggestions:

      <ul className="dashed">
        <li>
          <h4>Element template runtime versions</h4>
          Define the runtime version for each element template to ensure compatibility with the Camunda runtime.
        </li>
        <li>
          <h4>Zeebe user task is now Camunda user task</h4>
          Implement user tasks as Camunda user tasks to ensure full compatibility.
        </li>
        <li>
          <h4>Improved FEEL support</h4>
          Access script task result expression easily thanks to context keys suggestions.
        </li>
        <li>
          <h4>Document preview component in Forms</h4>
          Model and view uploaded documents in a new preview component.
        </li>
        <li>
          <h4>Bug fixes and more</h4>
          This release includes bug fixes and additional minor improvements to enhance your experience.
        </li>
      </ul>

### Checklist

To ensure you provided everything we need to look at your PR:

* [ ] __Brief textual description__ of the changes present
* [ ] __Visual demo__ attached
* [ ] __Steps to try out__ present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [ ] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

**Comment by jarekdanielak** [comment]
Looks good 👍

**Review by jarekdanielak — APPROVED** [review]

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,16 +44,20 @@ export function ReleaseInfo(props) {
...
-          <h4>Support for task listeners in Camunda 8</h4>
-          Define user task listeners in your Camunda 8.7 diagrams, with helpful guidance provided every step of the way.
+          <h4>Element template runtime versions</h4>
+          Define the runtime version for each element template to ensure compatibility with the Camunda runtime.
         </li>
         <li>
-          <h4>Enhanced Variables panel</h4>
-          Quickly access the Variables panel from the application footer or toggle it using a keyboard shortcut.
+          <h4>Zeebe user task is now Camunda user task</h4>
+          Implement user tasks as Camunda user tasks to ensure full compatibility.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:52]: Consider what we ship here:

* There is a clear path forward (Zeebe -> Camunda) user task first
* We now support you in the transition

I think this is not clearly reflected with the current phrase.

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -44,16 +44,20 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Support for task listeners in Camunda 8</h4>
-          Define user task listeners in your Camunda 8.7 diagrams, with helpful guidance provided every step of the way.
+          <h4>Element template runtime versions</h4>
+          Define the runtime version for each element template to ensure compatibility with the Camunda runtime.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:48]:
```suggestion
          Define runtime compatibility for your element templates to promote safe use across different Camunda versions.
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 3]
```diff
@@ -44,16 +44,20 @@ export function ReleaseInfo(props) {
...
         <li>
-          <h4>Enhanced Variables panel</h4>
-          Quickly access the Variables panel from the application footer or toggle it using a keyboard shortcut.
+          <h4>Zeebe user task is now Camunda user task</h4>
+          Implement user tasks as Camunda user tasks to ensure full compatibility.
         </li>
         <li>
-          <h4>Improved on-canvas search</h4>
-          Locate elements and details more efficiently within BPMN diagrams, thanks to upgraded search and filtering capabilities.
+          <h4>Improved FEEL support</h4>
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:55]: Improve 
```suggestion
          <h4>Improved editor suggestions</h4>
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 4]
```diff
@@ -44,16 +44,20 @@ export function ReleaseInfo(props) {
...
+          Access script task result expression easily thanks to context keys suggestions.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:56]: Be clear this relates to "editor intelligence", focused on Camunda 8:

```suggestion
          Editor intelligence now picks data provided by script tasks during suggestions.
```

Comment (barmac) [reply client/src/plugins/version-info/ReleaseInfo.js:56]: Makes sense

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 1 above)
Comment (barmac) [reply client/src/plugins/version-info/ReleaseInfo.js:52]: Another try:

```
<h4>Migration to Camunda user tasks</h4>
Implement user tasks managed by Camunda with guidance on the way.
```

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 2 above)
Comment (barmac) [reply client/src/plugins/version-info/ReleaseInfo.js:48]: Makes sense

**Comment by barmac** [comment]
Thanks @nikku for your suggestions. I merged them into the PR. Current:

image

**Review by nikku — APPROVED** [review]

**Comment by barmac** [comment]
I am merging this now but we can still adjust before the final release.
@Balazs-PM @crobbins215 @lmbateman

**Comment by lmbateman** [comment]
Sorry I missed this!

> — Zeebe user task is now Camunda user task

Should “Zeebe user task” and “Camunda user task” either be plural or in quotes? I think since we're referring to the name of a type of object, quotes would make sense.
- Zeebe user tasks are now Camunda user tasks
- “Zeebe user task” is now “Camunda user task”

Also, do we need to specify full compatibility with what?

> — Improved Camunda 8 editor intelligence
> Editor intelligence now picks data provided by script tasks.

I’m not quite sure what this means. But if no one else is confused, the grammar seems fine. :)

**Comment by barmac** [comment]
Thanks for the feedback.

Regarding the user tasks, perhaps we can rewrite the message (https://github.com/camunda/camunda-modeler/pull/4763#discussion_r1886663202). The changes in that area are:

* "Zeebe user task" got renamed to "Camunda user task"
* This is the preferred way of implementing user task.
* If you set the implementation to the other value, the application highlights this, and provides a message on deprecation.

> I’m not quite sure what this means.

The improvement is about suggesting the nested properties from script result:

https://github.com/user-attachments/assets/a7a6cd5a-6c3d-432b-bb5a-4e59b7c6da9a

**Comment by lmbateman** [comment]
For the user task update, how about:
— “Zeebe user task” renamed to “Camunda user task”

Then the next sentence is fine, but could also be something like your second sentence, “Camunda user tasks are now the preferred implementation for all user tasks.”

> Editor intelligence now picks data provided by script tasks.

Assuming I understand correctly, how about one of these?

- Nested properties are now provided in auto-complete results.
- Data provided by script tasks is now available in auto-completion.

If we want to stay with the original sentence, I would say “picks up” instead of “picks”, or perhaps use a different verb (sees? has access to?).

**Comment by barmac** [comment]
Thanks for the suggestions. Let's wait for feedback from the PMs, and we can adjust the release info after the winter holidays.

**Comment by crobbins215** [comment]
My suggestions 
    
    <h4>Enhanced FEEL support</h4>
    Easily access script task result expressions with the help of context key suggestions.

    <h4>Document preview component in Forms</h4>
    Model and preview uploaded documents with the new document preview component.

    <h4>Bug fixes and enhancements</h4>
     This release includes bug fixes and additional minor improvements to enhance your experience.

**Pull request merged**